trello_utils.py

```diff
@@ -4,24 +4,25 @@
 TRELLO_KEY = os.environ.get('TRELLO_KEY')
 TRELLO_TKN = os.environ.get('TRELLO_TKN')
 
-TRELLO_BOARD = '5f48fa1737fa77134ffe1e17'
+TRELLO_BOARD = os.environ.get('TRELLO_BOARD')
 TRELLO_TODO_LIST = '5f48fa17b70abe80f7f0942d'
 TRELLO_DONE_LIST = '5f48fa17d5f15e744c86a725'
```

Comment:
These board, todo_list and done_list variables are unused. Delete them

Reply:
TRELLO_BOARD actually is, in app.py, but good spot on the others.